libs/langchain-community/src/document_loaders/web/cheerio.ts

```diff
@@ -92,11 +99,13 @@ export class CheerioWebBaseLoader
     caller: AsyncCaller,
     timeout: number | undefined,
     textDecoder?: TextDecoder,
-    options?: CheerioOptions
+    options?: CheerioOptions,
+    headers?: Headers
```

Comment:
Can we combine these? Something like this:

```typescript
static async _scrape(
  ...other args...
  options?: CheerioOptions & { headers?: Headers },
) {
  const { headers, ...cheerioOptions } = options ?? {};
  ...existing code
}
```

Reply:
updated